src/website/views/components/avatar.pug

```diff
@@ -0,0 +1,323 @@
+      <div class="a-avatar a-icon -orange">
+          <svg>
+              <use xlink:href="#icon-user"></use>
+          </svg>
+      </div>
+
+h4 Size
+
+p.-text
+  | All three versions support diferent sizes including one of the classes: <code>-xs</code>, <code>-sm</code>,
+  | <code>-sm2</code>, <code>-sm3</code>, <code>-md</code>, <code>-lg</code>, <code>-xl</code>, <code>-xxl</code>.
+  | <code>-md</code> is the default size.
+ul#a-tabs--sizes.a-tabs
+  li.-active
+    a(href="#sizedLetters") Letters
+  li
+    a(href="#sizedIcons") Icons
+  li
+    a(href="#sizedImages") Images
+
+.example.-mb3
+  #sizedLetters.a-tabs-panel.-active
+    .-p3
+      .a-grid.-align-items--end
+        each size in sizes
+          div(class=`a-col -w${columnWidths[size]} -textCenter`)
+            div(class=`a-avatar -${size}`) AA
+            p.-text= size
+    :code(lang="html")
+      <div class="a-avatar -xs">AA</div>
+      <div class="a-avatar -sm">AA</div>
+      <div class="a-avatar -sm2">AA</div>
+      <div class="a-avatar -sm3">AA</div>
+      <div class="a-avatar -md">AA</div>
+      <div class="a-avatar -lg">AA</div>
+      <div class="a-avatar -xl">AA</div>
+      <div class="a-avatar -xxl">AA</div>
+
+  #sizedIcons.a-tabs-panel
+    .-p3
+      .a-grid.-align-items--end
+        each size in sizes
+          div(class=`a-col -w${columnWidths[size]} -textCenter`)
+            div(class=`a-avatar a-icon -${size}`)
+              svg
+                use(xlink:href=`#icon-user`)
+            p.-text= size
+
+    :code(lang="html")
+      <div class="a-avatar a-icon -xs">
+        <svg>
+          <use xlink:href="#icon-user"></use>
+        </svg>
+      </div>
+      <div class="a-avatar a-icon -sm">
+        <svg>
+          <use xlink:href="#icon-user"></use>
+        </svg>
+      </div>
+      <div class="a-avatar a-icon -sm2">
+        <svg>
+          <use xlink:href="#icon-user"></use>
+        </svg>
+      </div>
+      <div class="a-avatar a-icon -sm3">
+        <svg>
+          <use xlink:href="#icon-user"></use>
+        </svg>
+      </div>
+      <div class="a-avatar a-icon -md">
+        <svg>
+          <use xlink:href="#icon-user"></use>
+        </svg>
+      </div>
+      <div class="a-avatar a-icon -lg">
+        <svg>
+          <use xlink:href="#icon-user"></use>
+        </svg>
+      </div>
+      <div class="a-avatar a-icon -xl">
+        <svg>
+          <use xlink:href="#icon-user"></use>
+        </svg>
+      </div>
+      <div class="a-avatar a-icon -xxl">
+        <svg>
+          <use xlink:href="#icon-user"></use>
+        </svg>
+      </div>
+  #sizedImages.a-tabs-panel
+    .-p3
+      .a-grid.-align-items--end
+        each size in sizes
+          div(class=`a-col -w${columnWidths[size]} -textCenter`)
+            div(class=`a-avatar -${size}`)
+              img(src='../../assets/images/avatar.jpg')(alt="avatar")
+            p.-text= size
+    :code(lang="html")
+      <div class="a-avatar a-icon -xs">
```

Comment:
Remove a-icon class from all the examples under the Images tab. Not necessary for image avatars right?